santiago.rb

```diff
@@ -7,38 +7,42 @@
 pa3_answers = ["left", "painting", "right"]
 game_step = "intro"
 doll = false
-puts "You find yourself waking up in what appears to be an empty bedroom.\nNothingbut a bed, dresser and mirror.\nWhich would you like to inspect? \n> Mirror\n> Bed\n> Dresser"
+
+def ask_player question, options
+  puts question
+  options.each do |option|
+    puts "> #{option}"
+  end
+  print "> "
+  gets.chomp.downcase
+end
+
```

Comment:
This should be two methods, what if you tried something like 
```
def describe statement
  puts statement
end
```

what do you think?

Reply:
leave me alone